modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1728,9 +1776,33 @@ private void _validateValues(
 			return;
 		}
 
-		String dbType = objectField.getDBType();
+		if (StringUtil.equals(
```

Comment:
I was able to save one tab by:
```
DLFileEntry dlFileEntry = _dlFileEntryLocalService.fetchDLFileEntry(
	GetterUtil.getLong(entry.getValue()));

if (dlFileEntry != null) {
	_validateFileExtension(
		dlFileEntry.getExtension(), objectField.getObjectFieldId(),
		objectField.getName());
	_validateFileSize(
		dlFileEntry.getSize(), objectField.getObjectFieldId(),
		objectField.getName());

	return;
}

if (objectField.isRequired()) {
	throw new ObjectEntryValuesException.Required(
		objectField.getName());
}
```
Do you think it's worth changing?

Reply:
I applied the change, thanks!